compiler/native/expand.go

```diff
@@ -55,7 +55,7 @@ func (c *client) ExpandSteps(s yaml.StepSlice, tmpls map[string]*yaml.Template)
 		}
 
 		// parse source from template
-		src, err := c.Github.Parse(tmpl.Source)
+		src, err := c.Github.Parse(tmpl.Source, c.repo.GetBranch())
```

Comment:
we're grabbing default branch from the repo in the current context and passing it along to `Parse()`